### DISCUSSION TITLE
Location sudo password

### DISCUSSION THREAD

**Discussion by evilcarrotoverlord** [SteamDeckHomebrew/decky-loader#276] [Q&A] [answered]
So i set my password and seem to have forgotten it.
As some plugins require root i assume decky has stored it somewhere.
So my question is where wouldn't be stored?
Or would it be possible to make a plugin that would print it out or alow changing it using deckys root privilege?

**Comment by TrainDoctor** [comment, marked as answer]
Decky does not store your password and cannot do so. A plugin that would find the password and read it out would be utilizing a major security flaw and would not be allowed.
